src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkProvider.ts

```diff
@@ -134,7 +140,8 @@ export class TerminalWordLinkProvider extends TerminalBaseLinkProvider {
 		);
 	}
 
-	private async _activate(link: string) {
+	private async _activate(link: string, y: number) {
+		const pathSeparator = (isWindows ? '\\' : '/');
```

Comment:
Guess this was an existing bug, but this needs to be the backend's OS, not the frontend